schema-sandbox/schema.json

```diff
@@ -47,6 +77,38 @@
         "authors"
     ],
     "definitions": {
+        "date": {
+            "format": "date",
+            "pattern": "[0-9]{4}-[0-1]{1}[0-9]{1}-[0-3]{1}[0-9]{1}",
+            "type": "string"
+        },
+        "doi": {
+            "pattern": "^10\\.\\d{4,9}(\\.\\d+)?/[A-Za-z0-9-\\._;\\(\\)\\[\\]\\\\\\\\:/]+$",
```

Comment:
the end of the regex looks odd, maybe we should simplify

Reply:
BTW I generated this string from the existing one in the YAML

Reply:
Yes, looks like there may be some escaping madness going on. Also, see [my comment here](https://github.com/citation-file-format/citation-file-format/pull/123#discussion_r632392685), perhaps we should drop the pattern here altogether?

Reply:
I think we do need some pattern for dois, because people tend to want to enter URLs.

Reply:
I've created #129, so this is not blocking for this PR